ocw/lib/provider.py

```diff
@@ -13,6 +15,23 @@ def __init__(self, namespace: str):
         self._namespace = namespace
         self.dry_run = PCWConfig.getBoolean('default/dry_run')
         self.logger = logging.getLogger(self.__module__)
+        self.auth_json = self.read_auth_json()
+
+    def read_auth_json(self):
+        authcachepath = Path('/tmp/pcw/{}/{}/auth.json'.format(self.__class__.__name__, self._namespace))
```

Comment:
Storing any auth credentials in `/tmp` that could be deleted at any time by anyone and is also lacking any protection (`/tmp` is typically drwxrwxrwt) might give us headache soon. Also when thinking of containers we probably need to mount this directory, so we can pass credential files to the container.
Could we perhaps switch the path from `/tmp/pcw` to `/var/pcw` (or something else) so that we have a canonical path for the credentials and avoid all associated risks that `/tmp` has?

```suggestion
        authcachepath = Path('/var/pcw/{}/{}/auth.json'.format(self.__class__.__name__, self._namespace))
```